modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -75,9 +65,9 @@ void dataLoad(std::istream & stream, FeatureFloodCount::FeatureData & feature, v
 }
 
 template<>
-void dataLoad(std::istream & stream, MooseSharedPointer<FeatureFloodCount::FeatureData> & feature, void * context)
+void dataLoad(std::istream & stream, std::unique_ptr<FeatureFloodCount::FeatureData> & feature, void * context)
```

Comment:
wait... why is this necessary? in DataIO.h you added 

```
void dataLoad(std::istream & stream, std::unique_ptr<T> & feature, void * context);
```
￼
This speacialization does nothing more than the generic template.